bitmap-annashivwill/index.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+const maps = require(`${__dirname}/model/read-write.js`);
+const transforms = require(`${__dirname}/lib/transforms.js`);
+
+maps.readWrite();
```

Comment:
@William-Skelton @shivprogrammer @annaul excellent job keeping your `index` file clean - no need to use `${__dirname}` in your require paths - relative file paths are fine for this